### DISCUSSION TITLE
When the ACInterface module changes Kappa, why does it not converge？

### DISCUSSION THREAD

**Discussion by biaogxb** [idaholab/moose#22664] [Q&A Modules: Phase field] [answered]
I carefully read the snow.i example and found that the ‘InterfaceOrientationMaterial’ is used to calculate the kappa（eps in that article） value I need, 
![1668136586348](https://user-images.githubusercontent.com/95063954/201255388-09711f79-5ac4-462f-bfc7-44d9dd8af27f.jpg)
so I changed the code for calculating ‘ACInterface’ as follows：
```
'[./anisoACinterface] type = ACInterface variable = eta  kappa_name=eps [../]'
'[./material] type = InterfaceOrientationMaterial op = eta [../]'
```
I don't know why my calculation is not convergent. Can't I directly fill ‘eps’ into ACInterface as ‘kappa’？
Here is my input file：
[input.txt](https://github.com/idaholab/moose/files/9986235/input.txt)
Can someone help me? Thank you！

**Comment by dschwen** [comment, marked as answer]
What is `eps`?

**Reply by biaogxb** [reply]
'eps' represents the anisotropic interface parameter in the 'modules/phase_field/src/materials/InterfaceOrientationMaterial.C' source code.（？）
![image](https://user-images.githubusercontent.com/95063954/201258997-1f97a26f-7ac3-42b1-8319-abf13f01b9e3.png)
![image](https://user-images.githubusercontent.com/95063954/201259228-ed96d055-0bff-4ea6-8389-4a761b5af578.png)

I'm not sure if this is the way to couple the anisotropic interface parameter（eps in InterfaceOrientationMaterial.C）into the ACInterface. Do you have any suggestions？
I just want to set the kappa in the ACInterface as shown in the following figure：
![1668136586348](https://user-images.githubusercontent.com/95063954/201258876-241d48ae-086a-44f2-9722-323bba1970f2.jpg)
![image](https://user-images.githubusercontent.com/95063954/201259897-37136fb9-7b9c-4d65-a2c9-a456adb26fa8.png)

Do you have any suggestions？

**Reply by GiudGiud** [reply]
so is eps supposed to be the same as kappa? Pardon my ignorance.

I would start from a working example, then make the minimal amounts of changes by replacing the object that defines kappa with a slightly modified version

**Reply by biaogxb** [reply]
Ok, thank you. If there is any progress, I hope you can remind me. Thank you very much！

**Reply by GiudGiud** [reply]
I won’t be working on this @biaogxb . Please feel free to let us know your progress with this
